lib/logstash/outputs/bson/big_decimal.rb

```diff
@@ -0,0 +1,66 @@
+# Copyright (C) 2009-2014 MongoDB Inc.
```

Comment:
sure, I don't have either the knowledge to decide if should stay or can be removed. @suyograo @jordansissel can you chime in ? what do you think?